creator-node/src/routes/tracks.js

```diff
@@ -300,37 +300,7 @@ module.exports = function (app) {
 
       // if track created, ensure files exist with trackuuid = null and update them.
       if (trackCreated) {
-        // Update the transcoded 320kbps copy
-        if (transcodedTrackUUID) {
-          const transcodedFile = await models.File.findOne({
-            where: {
-              fileUUID: transcodedTrackUUID,
-              cnodeUserUUID,
-              trackUUID: null,
-              type: 'copy320'
-            },
-            transaction: t
-          })
-          if (!transcodedFile) {
-            throw new Error('Did not find a transcoded file for the provided CID.')
-          }
-          const numAffectedRows = await models.File.update(
-            { trackUUID: track.trackUUID },
-            { where: {
-              fileUUID: transcodedTrackUUID,
-              cnodeUserUUID,
-              trackUUID: null,
-              type: 'copy320'
-            },
-            transaction: t
-            }
-          )
-          if (numAffectedRows === 0) {
-            throw new Error('Failed to associate the transcoded file for the provided track UUID.')
-          }
-        }
-
-        // Update the corresponding segment files
+        // Associate created trackUUID to each segment file
         const trackFiles = await models.File.findAll({
           where: {
             multihash: trackSegmentCIDs,
```

Comment:
does this do an implicit IN with these cids? i'm assuming it has to, otherwise this wouldn't have ever worked

Reply:
it must lol, i didn't even change this part!